meguca.user.js

```diff
@@ -117,6 +117,10 @@ function insertCuteIntoCSS() {
         ".dangerous_roll {font-size: 110%; color: #f00000; }" +
         ".dead_fuck { color: #e55e5e; }" +
         ".decision_roll { animation: decision_blinker 0.4s linear 2; color: lightgreen; } @keyframes decision_blinker { 50% { color: green } }" +
+        ".planeptune_wins { animation: planeptune_blinker 0.6s linear " + getIterations(0.4) + "; color: #fff; } @keyframes planeptune_blinker { 50% { color: mediumpurple} }"+

```

Comment:
>0.6s linear " + getIterations(0.4)

Can you change that to `getIterations(0.6)`, so it with flash for the correct amount of time.